ios/AppcuesReactNative.swift

```diff
@@ -88,4 +93,63 @@ class AppcuesReactNative: NSObject {
             resolve(implementation.didHandleURL(url))
         }
     }
+
+    // MARK: Event Emitting
+
+    override func supportedEvents() -> [String]! {
+        [Self.eventName]
+    }
+
+    override func startObserving() {
+        hasListeners = true
+    }
+
+    override func stopObserving() {
+        hasListeners = false
+    }
+}
+
+extension AppcuesReactNative: AppcuesAnalyticsDelegate {
+    func didTrack(analytic: AppcuesAnalytic, value: String?, properties: [String: Any]?, isInternal: Bool) {
+        guard hasListeners else { return }
+
+        let analyticName: String
+        switch analytic {
+        case .event:
+            analyticName = "EVENT"
+        case .screen:
+            analyticName = "SCREEN"
+        case .identify:
+            analyticName = "IDENTIFY"
+        case .group:
+            analyticName = "GROUP"
+        }
```

Comment:
nice catch getting these formatted consistently across platforms!

Reply:
I was using the numeric ordinal/rawValue except they're ordered differently 😅 and the named strings are more dev-friendly anyways.